Doc fix: Correct hubble exporter config lines

saintdle opened the pull request:
Fixes Hubble exporter doc where incorrect case is used on some of the config lines to configure the feature. This incorrect case leads to errors in the cilium-agent and the files cannot be wrote. 

Added information on how to disable the static exporter config #30425

AwesomePatrol commented on Documentation/observability/hubble-exporter.rst:
```diff
@@ -235,8 +235,8 @@ You can change flow log settings without a need for pod to be restarted
-      --set hubble.export.dynamic.config.content[0].IncludeFilters[0].source_pod[0]='kube_system/*' \\
-      --set hubble.export.dynamic.config.content[0].IncludeFilters[1].destination_pod[0]='kube_system/*'
+      --set hubble.export.dynamic.config.content[0].includeFilters[0].source_pod[0]='kube_system/*' \\
+      --set hubble.export.dynamic.config.content[0].includeFilters[1].destination_pod[0]='kube_system/*'
```
We can use this opportunity to also fix asterisk (`*`). Either remove it (as this filter prefix matches ns+pod_name) or escape it (`\*`) so the lower line won't be in italics.

saintdle replied:
done

learnitall commented on Documentation/observability/hubble-exporter.rst:
```diff
@@ -47,7 +47,13 @@ Verify that flow logs are stored in target files:
-Now you can configure your logging solution to consume logs from your Hubble export file path.
+Once you have configured the Hubble Exporter, you can configure your logging solution to consume logs from your Hubble export file path.
+
+To disable the static configuration, you must remove the ``hubble-export-file-path`` key in the ``cilium-config`` ConfigMap and manually clean up the log files created in the specified location in the container. The below command will restart the Cilium pods, if you edit the ConfigMap manually, you will need to restart the Cilium pods.
```
Small punctuation suggestion. I feel like this one sentence could be split into two:

```suggestion
To disable the static configuration, you must remove the ``hubble-export-file-path`` key in the ``cilium-config`` ConfigMap and manually clean up the log files created in the specified location in the container. The below command will restart the Cilium pods. If you edit the ConfigMap manually, you will need to restart the Cilium pods.
```

learnitall reviewed (changes requested):
One small punctuation nit. Also, would you mind wrapping your lines? I'm not sure if we have a specific max line length, but it would be great if we could keep each line to a maximum of around 90 to 100 characters.

Otherwise LGTM! Thanks!

learnitall commented on Documentation/observability/hubble-exporter.rst:
```diff
@@ -47,7 +47,13 @@ Verify that flow logs are stored in target files:
+Once you have configured the Hubble Exporter, you can configure your logging solution to consume logs from your Hubble export file path.
+
+To disable the static configuration, you must remove the ``hubble-export-file-path`` key in the ``cilium-config`` ConfigMap and manually clean up the log files created in the specified location in the container. The below command will restart the Cilium pods. If you edit the ConfigMap manually, you will need to restart the Cilium pods.
+
```
Here's an example of line wrap:

```suggestion
Once you have configured the Hubble Exporter, you can configure your logging solution to consume
logs from your Hubble export file path.

To disable the static configuration, you must remove the ``hubble-export-file-path`` key in
the ``cilium-config`` ConfigMap and manually clean up the log files created in the specified
location in the container. The below command will restart the Cilium pods. If you edit the
ConfigMap manually, you will need to restart the Cilium pods.
```

learnitall reviewed (approved):
Thanks!

saintdle committed:
docs: Add information to disable Hubble Exporter static config

Fixes: #30425

Signed-off-by: Dean <22192242+saintdle@users.noreply.github.com>

saintdle committed:
docs: Remove erroneous line from Dynamic Hubble Exporter example

This commit removes a line that was unnecessarily copied from the
static config example into the dynamic config example.

Signed-off-by: Dean <22192242+saintdle@users.noreply.github.com>

saintdle committed:
docs: Correct Hubble Exportor config lines in dynamic example

This commit makes corrections to the Hubble Exporter dynamic example.
This includes lowercasing the "I" in `IncludeFilters`, as well as
removing unnecessary `*`s in the provided filters.

Signed-off-by: Dean <22192242+saintdle@users.noreply.github.com>
